src/main/java/teammates/ui/automated/AutomatedAction.java

```diff
@@ -58,10 +59,36 @@ protected String getRequestParamValue(String paramName) {
         return HttpRequestHelper.getValueFromRequestParameterMap(request, paramName);
     }
     
+    /**
+     * Retrieves the value for the specified parameter expected to be present in the http request.
+     *
+     * @param paramName  a constant from the {@link Const.ParamsNames} class.
+     * @return the value of the specified parameter.
```

Comment:
This line can be avoided (to reduce duplication). Instead, rephrase the the overview sentence to `Returns the value ...`